pkg/eventsourcing/storage/store_posgres_config.go

```diff
@@ -15,13 +15,12 @@
 package storage
```

Comment:
@jastBytes there's a typo in the file name. a `t` missing

Reply:
Will rename.